components/brave_shields/browser/brave_resource_dispatcher_host_delegate.cc

```diff
@@ -30,6 +38,31 @@ void BraveResourceDispatcherHostDelegate::AppendStandardResourceThrottles(
     content::ResourceContext* resource_context,
     ResourceType resource_type,
     std::vector<std::unique_ptr<content::ResourceThrottle>>* throttles) {
+
+  CHECK(g_brave_browser_process->tracking_protection_service()->IsInitialized());
```

Comment:
wy are we doing this in AppendStandardResourceThrottles? This code isn't doing anything with resource throttles